packaging/package.sh

```diff
@@ -48,6 +48,13 @@ function packaging() {
     fi
 
     DISTRO=$3
+
+    JDK=$(rpm -qa | grep "java-1...0-openjdk-devel")
+    for version in $JDK; do [ "${version:7:1}" -ge "7" ] && echo  ${version:7:1}; done;
+    if [ "$?" -gt "0" ] || [ -z "$JDK" ] ; then
```

Comment:
My previous comment is most likely related to this. If the last grep line doesn't match -ge 7, it'll fail